datafusion/wasmtest/README.md

```diff
@@ -37,9 +37,12 @@ First, [install wasm-pack](https://rustwasm.github.io/wasm-pack/installer/)
 Then use wasm-pack to compile the crate from within this directory
 
 ```
+export RUSTFLAGS="--cfg tokio_unstable"
 wasm-pack build
 ```
 
+**Note:** Due to limited support for WASM in `tokio`, setup `RUSTFLAGS` to ensure that all features enabled in the workspace can be compiled.
```

Comment:
🤔 I am not sure what the implication of trying to compile tokio with this flag is. 

I think it is important that we make it easy for people to use DataFusion with WASM and so requiring a special compiler flag is not ideal.

I assume this is needed because one of the features (perhaps `"fs"`) is not supported by wasm by default?

Perhaps we can remove that feature from the workspace dependency and only activate it when necessary?

Reply:
I will adopt @jonmmease 's suggestion.